Deploy heart sticker to Heroku server to test SnapKit custom sticker pipeline

uci-mars opened the pull request:
Available route:
` /heartbeat/:option/:beats/:type/:goal`

Examples:
```
/heartbeat/chartjunk-domain-relevant-1/90/pulse/200
/heartbeat/plain-domain-relevant-1/85/pulse/0
/heartbeat/plain-domain-relevant-1/80/shake/0
/heartbeat/plain-domain-relevant-2/80/pulse/0
/heartbeat/plain-domain-relevant-2/80/shake/0
```

Commit:
ccapture + pixi + gsap test (very buggy)

Commit:
experiment #2: capture gsap animation frames and use gifshot to encode gif

gif can be exported but still facing with no transparency background issue.
need to figure out how to do text overlay

Commit:
experiment #3: Gif rendered with gif.js and canvg (most successful attempt yet)

Capture each animation frame as svg, convert to png using canvg, and gif.js to encode transparent frames into gif

Commit:
opened a route for heartbeat sticker

Commit:
organize heartbeat animation files, added a route on express server to generate a gif response of the sticker, modified gif.js to accept svg to render animation, implement plain-domain-relevant-2 sticker

Commit:
implemented a semaphore to render gif when every frames are fully loaded (successfully removed the black frame Search Results Web results  🎉) -- included swipe animation and chartjunk w/ goals sticker

depstein commented:
This looks pretty clean, I'm happy to merge it in. Two comments:
- Does the heartbeat_render.js only apply to heartbeats, or can that same code generalize to other domains? More of a naming question, it looks like we should be able to use the same code once we start creating other stickers as well. There's just a few places where "heart" is actually referenced, and these feel like they could be pretty easily replaced with "sticker".
- The code could do with more comments to explain what's happening at each step and why. Given enough time I could probably figure out what's going on, but there's lots of weird parsing things and library calls that would be helpful to label so I (or someone else) could understand their role.

Commit:
A WHOLE LOT of code refactoring, implement count animation

- Figured out how to implement the count animation using GSAP, which needed me to rewrite the gif generating code
- Apparently, we can use document.getElementbyId() to change the value of a <text> element inside a svg (wish I figured this out sooner)
- Refactor the code to make render.js and options.js reusable with all stickers
- Different type of stickers now exist in different html file
